amivapi/cli.py

```diff
@@ -90,7 +90,7 @@ def ldap_sync(config, sync_all, nethz):
             else:
                 for user in nethz:
                     if ldap.sync_one(user) is not None:
-                        echo("Succesfully synchronized '%s'." % user)
+                        echo("Successfully synchronized '%s'." % user)
```

Comment:
I think this is the most important change of this PR, well done 👍

Reply:
#proud